.github/workflows/audit.yml

```diff
@@ -4,7 +4,9 @@ on:
   push:
   schedule:
     - cron: "40 10 * * *"
-
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
```

Comment:
Just to double check `github.ref` is the pull request ID?

Reply:
That's the branch: https://docs.github.com/en/actions/learn-github-actions/environment-variables#default-environment-variables